server/services/nl.py

```diff
@@ -54,6 +54,9 @@ def pick_best(probs):
 
 def pick_option(class_model, q, categories):
   """Return the assigned label or Inconclusive."""
+  if not q:
+    return "Inconclusive"
```

Comment:
Should this make this an Enum or just None? If None works, it would be better.

Reply:
That will require some more changes which I shouldn't touch in this PR. I will share a follow-up PR which should address those classification types and convert them to types and then we can use None

Reply:
follow up PR sg!

Reply:
Lets do beautification/refactors post Monday please.